package.json

```diff
@@ -22,6 +22,14 @@
   },
   "homepage": "https://github.com/Ian-Stewart-Binks/foosball#readme",
   "devDependencies": {
-    "express": "^4.13.3"
+    "express": "^4.13.3",
```

Comment:
@Anirudh94 I changed the dependencies and devDependencies. Can you do a quick double check for me?